features/bootstrap/FeatureContext.php

```diff
@@ -856,7 +856,14 @@ public function install_wp( $subdir = '' ) {
 			if ( $install_cache_path ) {
 				mkdir( $install_cache_path );
 				self::dir_diff_copy( $run_dir, self::$cache_dir, $install_cache_path );
-				self::run_sql( 'mysqldump --no-defaults', array( 'result-file' => "{$install_cache_path}.sql" ), true /*add_database*/ );
+
+				$support_column_statistics = exec( 'mysqldump --help | grep "column-statistics"' );
```

Comment:
We should stick to one way to locating the `mysqldump` binary only:
```suggestion
				$support_column_statistics = exec( '/usr/bin/env mysqldump --help | grep "column-statistics"' );
```